README.md

```diff
@@ -31,76 +31,55 @@ Execute the following commands to create a duplicate of a repository; you need t
     $ git clone https://github.com/exampleuser/new-respository.git
     $ cd new-repository
     $ heroku create
+    
+The output from Heroku should produce the domain name... which will be something like: `https://fathomless-thicket-78154.herokuapp.com/`. We will refer to this url as the environment variable MY_APP_URL.
 
 #### **Creating the App and Store Entry**
 
-To regsister the new app, use the [Developer Portal](https://dev-portal.onshape.com) to create your OAuth Application and private Store Entry, which you can then subscribe to in the [App Store](https://appstore.onshape.com) in order to add it to your documents. The output from Heroku should produce the domain name:
+To regsister the new app, use the [Developer Portal](https://dev-portal.onshape.com) to create your OAuth Application. Fill all the fields in, make sure to add "Application can read your documents" scope, and make sure to set the URLs to point to your app url correctly as shown here:
+    
+    Redirect URLs: <MY_APP_URL>/oauthSignin
+    iframe URL: <MY_APP_URL>/oauthRedirect
```

Comment:
I think this is wrong. Should it be
Redirect URLs: <MY_APP_URL>/oauthRedirect
iframe URL: <MY_APP_URL>/oauthSignin